lib/puppet/interface/documentation.rb

```diff
@@ -56,7 +56,8 @@ module TinyDocs
     # @dsl Faces
     attr_doc :summary do |value|
       value =~ /\n/ and
-        raise ArgumentError, "Face summary should be a single line; put the long text in 'description' instead."
+        #TRANSLATORS 'Face' refers to a programming API in Puppet
```

Comment:
I feel like you should add `and should not be translated` here. Also, `summary` and `description` are key values that we grab data from. I have no idea for sure, but I have a feeling those are going to need to be in English as well

Reply:
Ok, I agree

Reply:
Too bad we don't have a text decoration for words that won't need translation, would help with so many duplicated translator notes